src/flicking.js

```diff
@@ -1318,6 +1321,68 @@ export default class Flicking extends Mixin(Component).with(EventHandler) {
 	}
 
 	/**
+	 * Get current flicking status
+	 * @ko 현재의 상태 값을 반환한다.
+	 * @method eg.Flicking#getStatus
+	 * @param {Boolean} stringify Set true if want get stringified status value <ko>상태 값을 문자열로 전달받고자 하는지 여부</ko>
+	 * @return {{panel: {index: (*|number), no: (*|number), currIndex: number, currNo: number}, $list}}
+	 */
+	getStatus(stringify) {
+		const panel = this._conf.panel;
+		const rxStyle = /(transform|left|top|will-change|box-sizing|width):[^;]*;/g;
+		const status = {
+			// current panel position
+			panel: {
+				index: panel.index,
+				no: panel.no,
+				currIndex: panel.currIndex,
+				currNo: panel.currNo
+			},
+
+			// panel's html
+			$list: panel.$list.map(v => ({
+				style: v.style.cssText.replace(rxStyle, "").trim(),
```

Comment:
Thinking the whole process of getting and restoring will be like:
- on getting: instantiate -> some user action(or not) -> get current status of flicking element
- on restoring: instantiate -> restore the previous state with state data

Will not save whole css style, because on restoring the flicking element can stay differently than the previous (ex. get status when is vertical mode, and user changes the portrait mode of device).

To avoid that situation, keeping the instantiated style will be more reasonable, rather than overwriting with the previous.